dll/3rdparty/libjpeg/CMakeLists.txt

```diff
@@ -10,6 +10,8 @@ include_directories(
     ${REACTOS_SOURCE_DIR}/sdk/include/reactos/libs/libjpeg
     ${REACTOS_SOURCE_DIR}/sdk/include/reactos/libs/zlib)
 
+spec2def(libjpeg.dll libjpeg.spec ADD_IMPORTLIB)
```

Comment:
In the `add_module()` below, it's missing `${CMAKE_CURRENT_BINARY_DIR}/libjpeg.def`